docs/general-settings.md

```diff
@@ -12,9 +12,11 @@ settings: {
     favicon: 'path/to/favicon.ico'
   }
 }
-````
+```` 
+
 * **hideNavigation** disables Luigi's default out-of-the-box navigation when set to `true`. As a result, neither the left nor top navigation is visible in your application, and you can implement your own navigation UI. By default, the parameter is set to `false`, which means the navigation is enabled.
 * **backdropDisabled** prevents the backdrop layer from covering the top and left navigation when showing modal windows. By default, the backdrop is set to `true`.
 * **header.logo** defines the top left navigation logo. It has a fixed height of 28px.
 * **header.title** defines the top left navigation title.
 * **header.favicon** defines the favicon. It rquires a standard favicon file with the `.ico` extension, and 16x16px or 32x32px dimensions.
+* **responsiveNavigation** allows to add a button to the very left of the top navigation which will show or hide the left navigation when clicked. Possible values are `simple` and `simpleMobileOnly`. `simple` will display the button regardless of the browser window´s size, while `simpleMobileOnly` will show the button when the width is lower than _600px_. If `responsiveNavigation` is not specified with one of these values the button will not be displayed. The same applies when `hideNavigation` is enabled in [general settings](#general-settings), or hideNav is set in the current active navigation node. 
```

Comment:
```suggestion
* **responsiveNavigation** allows adding a button on the left side of the top navigation. Upon click, the button shows or hides the left navigation. The possible values are `simple` and `simpleMobileOnly`. `simple` displays the button regardless of the browser window´s size, while `simpleMobileOnly` shows the button when the width is lower than `600px`. If you don't specify any value for  **responsiveNavigation**, the button will not be displayed. The same applies when you enable **hideNavigation** in [general settings](#general-settings), or set it for the current active navigation node. 
```

Reply:
Counter proposal, just changing the end of the sentence because the rest sounds really great :)
`
... The same applies when you enable either **hideNavigation** in [general settings](#general-settings), or **hideNav** for the current active navigation node. 
`

Reply:
Yes, just realised that it's a different param. Looks good!